build-tools/packages/build-cli/src/commands/update/fluid-imports.ts

```diff
@@ -0,0 +1,332 @@
+/* eslint-disable max-depth */
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Package, type Logger } from "@fluidframework/build-tools";
+import { Flags } from "@oclif/core";
+import path from "node:path";
+import { Project } from "ts-morph";
+import { BaseCommand } from "../../base";
+
+type ApiLevel = "internal" | "public" | "alpha" | "beta";
+
```

Comment:
It would be good to have the scopes named throughout instead of a bunch of isolated strings.
Having a const array of them lets you process known/unknown easier and more reliably - will post another comment.
```suggestion
const publicLevel = "public";
const betaLevel = "beta";
const alphaLevel = "alpha";
const internalLevel = "internal";

const knownLevels = [publicLevel, betaLevel, alphaLevel, internalLevel] as const;

type ApiLevel = (typeof knownLevels)[number];

```

Reply:
Agreed, will do.